apps/explorer/src/ui/Banner.tsx

```diff
@@ -16,23 +18,54 @@ const bannerStyles = cva(
                 error: 'bg-issue-light text-issue-dark',
                 message: 'bg-sui-light text-hero',
             },
-            align: {
-                left: 'justify-start',
-                center: 'justify-center',
-            },
             fullWidth: {
                 true: 'w-full',
             },
+            spacing: {
+                md: 'px-3 py-2',
+                lg: 'p-5',
+            },
         },
         defaultVariants: {
             variant: 'message',
+            spacing: 'md',
         },
     }
 );
 
-export interface BannerProps extends VariantProps<typeof bannerStyles> {
+const contentStyles = cva(
+    'flex flex-1 items-center gap-2 max-w-full min-w-0 flex-nowrap',
+    {
+        variants: {
+            align: {
+                left: 'justify-start',
+                center: 'justify-center',
+            },
+        },
+        defaultVariants: {
+            align: 'left',
+        },
+    }
+);
+
+const closeBtnStyles = cva('self-start', {
+    variants: {
+        spacing: {
+            md: '-mt-1 -mr-2',
+            lg: '-mt-4 -mr-4',
+        },
+    },
+    defaultVariants: {
+        spacing: 'md',
+    },
+});
+
+export interface BannerProps
+    extends VariantProps<typeof bannerStyles>,
```

Comment:
Probably should also should extend `VariantProps<typeof closeBtnStyles>`, even if it does overlap from bannerStyles.